coderd/organizations_test.go

```diff
@@ -140,3 +140,144 @@ func TestPostOrganizationsByUser(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestPatchOrganizationsByUser(t *testing.T) {
+	t.Parallel()
+	t.Run("Conflict", func(t *testing.T) {
+		t.Parallel()
+		client := coderdtest.New(t, nil)
+		user := coderdtest.CreateFirstUser(t, client)
+
+		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
+		defer cancel()
```

Comment:
You can do this for the other tests too
```suggestion
     ctx := testutil.Context(t, testutil.WaitLong)
```